pipeline/outputs/nats.md

```diff
@@ -65,3 +67,7 @@ Each record is an individual entity represented in a JSON array that contains a
 ]
 ```
 
+Please note that this module writes to the NATS stream as JSON, rather than MsgPack ( msgpack_to_json ).
+
+
+See also: https://github.com/fluent/fluent-bit/blob/master/plugins/out_nats/nats.c#L185C1-L189C49
```

Comment:
```suggestion
{% hint style="info" %}
**Note:** This module
[writes to the NATS stream as JSON](https://github.com/fluent/fluent-bit/blob/master/plugins/out_nats/nats.c#L185C1-L189C49),
rather than MsgPack (`msgpack_to_json`).
{% endhint %}
```